tsconfig.json

```diff
@@ -35,5 +35,5 @@
     "isolatedModules": true
   },
   "exclude": ["./dist/**/*", "./node_modules/**/*"],
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"]
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "src/components", "src/custom.d.ts"]
```

Comment:
I thought the blob `**.*/ts` would include `src/custom.d.ts` & `src/components`. Does it not?

Reply:
fixed.